.gitignore

```diff
@@ -6,4 +6,5 @@ __pycache__
 
 upstream
 
-.terraform*
\ No newline at end of file
+.terraform*
+terraform.tfstate*
```

Comment:
they will be able to see it, however it will not be committed

since tfstate stores sensitive info unencoded it is not best practice to commit it either

to persist state not locally s3 would be a better option

Reply:
Okay, makes sense